simphony/core/cuba.yml

```diff
@@ -474,3 +474,52 @@
   number: 67
   shape: [1]
   type: double
+- description: Point item identifier
+  domain: [ATM, DEM, FEM, FVM, LBM, SPH, VIS]
```

Comment:
Agree, but I am not aware of exactly which types are correct for each domain.
I can figure out some of them by looking at the wrappers but I'm not sure this is the best way to proceed. 

It would be nice if the information is updated in the wiki: 

https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/API_proposal:_CUDS_containers_and_number_of_items 

or 

https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/API_proposal:_CUDS_containers_and_number_of_items